### PR TITLE
Don't add accessors for a property with an arrow-body

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Fixes https://github.com/dotnet/roslyn/issues/6034

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
Tagging @dotnet/roslyn-ide

File: src/EditorFeatures/CSharpTest/CodeActions/ReplaceMethodWithProperty/ReplaceMethodWithPropertyTests.cs [hunk 1]
```diff
@@ -35,7 +35,7 @@ public void TestMethodWithArrowBody()
         {
             Test(
 @"class C { int [||]GetFoo() => 0; }",
-@"class C { int Foo { get; } => 0; }");
+@"class C { int Foo => 0; }");
```
Comment (dpoeschl) [inline src/EditorFeatures/CSharpTest/CodeActions/ReplaceMethodWithProperty/ReplaceMethodWithPropertyTests.cs]: Want to add a WorkItem to this pointing to 6034?

**Comment by dpoeschl** [comment]
:+1:

File: src/EditorFeatures/CSharpTest/CodeActions/ReplaceMethodWithProperty/ReplaceMethodWithPropertyTests.cs (hunk 1 above)
Comment (CyrusNajmabadi) [inline src/EditorFeatures/CSharpTest/CodeActions/ReplaceMethodWithProperty/ReplaceMethodWithPropertyTests.cs]: Sure!  What do we use the workitems for BTW?  I noticed a bunch of them are just random numbers.  And I don't even know what bug DB they're even referring to...

**Comment by tmeschter** [comment]
LGTM

**Comment by balajikris** [comment]
:+1:

**Comment by CyrusNajmabadi** [comment]
retest this please

**Comment by CyrusNajmabadi** [comment]
retest prtest/lin/dbg/unit32 please

**Comment by DustinCampbell** [comment]
@Pilchie, should this be ported to stabilization for 1.1?

**Comment by Pilchie** [comment]
Yes, I think so, since it's a pretty bad intersection of two "new" features.

**Pull request merged**